### PR TITLE
Update namespace descriptions.

### PR DESCRIPTION
**Pull request by natke**
## Update namespace descriptions for the ML.NET API documentation

Preview links:
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml?branch=pr-en-us-90&view=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.data?branch=pr-en-us-90&view=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.trainers?branch=pr-en-us-90&view=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.trainers.fasttree?branch=pr-en-us-90&view=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.trainers.lightgbm?branch=pr-en-us-90&view=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.trainers.recommender?branch=pr-en-us-90&view=ml-dotnet-preview&viewFallbackFrom=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms?branch=pr-en-us-90&view=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.text?branch=pr-en-us-90&view=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.image?branch=pr-en-us-90&view=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.onnx?branch=pr-en-us-90&view=ml-dotnet-preview&viewFallbackFrom=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.tensorflow?branch=pr-en-us-90&view=ml-dotnet-preview&viewFallbackFrom=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.timeseries?branch=pr-en-us-90&view=ml-dotnet-preview&viewFallbackFrom=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.samplesutils?branch=pr-en-us-90&view=ml-dotnet-preview&viewFallbackFrom=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.staticpipe?branch=pr-en-us-90&view=ml-dotnet-preview&viewFallbackFrom=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.calibrators?branch=pr-en-us-90&view=ml-dotnet
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.runtime?branch=pr-en-us-90&view=ml-dotnet

### CODE REVIEW

File: dotnet/xml/ns-Microsoft.ML.Transforms.Image.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <Namespace Name="Microsoft.ML.Transforms.Image">
   <Docs>
     <summary>
-      Namespace containing image loading and <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/glossary#transform">transformation</a> capabilities. 
-      It is contained in the <a href="https://www.nuget.org/packages/Microsoft.ML.ImageAnalytics/">Microsoft.ML.ImageAnalytics nuget package.</a>
+      Namespace containing image loading and transforming components.
```
Comment (shmoradims) [inline dotnet/xml/ns-Microsoft.ML.Transforms.Image.xml]: >loading and transforming [](start = 33, length = 24)

loading and transformation

File: dotnet/xml/ns-Microsoft.ML.Runtime.xml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <Namespace Name="Microsoft.ML.Runtime">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Namespace containing advanced types that support ML.NET.</summary>
```
Comment (sfilipi) [inline dotnet/xml/ns-Microsoft.ML.Runtime.xml]: >Namespace containing advanced types that support ML.NET [](start = 13, length = 55)

it is mostly things related to IHost, IChannel: logging, setting the seed. 

Maybe Namespace containing types needed by the ML.NET runtime. 

Advanced types makes me thing advanced ML things.

File: dotnet/xml/ns-Microsoft.ML.SamplesUtils.xml [hunk 3]
```diff
@@ -1,8 +1,10 @@
 <Namespace Name="Microsoft.ML.SamplesUtils">
   <Docs>
     <summary>
-      Namespace containing utilities to download and featurize datasets for the API reference samples.
-      It is part of the Microsoft.ML.SampleUtils nuget package.
+    Namespace containing utilities to download and featurize datasets for the API reference samples.
     </summary>
+    <remarks>
+    Included in the Microsoft.ML.SampleUtils nuget package.
```
Comment (sfilipi) [inline dotnet/xml/ns-Microsoft.ML.SamplesUtils.xml]: >Microsoft.ML.SampleUtils [](start = 20, length = 24)

link to: [https://www.nuget.org/packages/Microsoft.ML.SampleUtils](https://www.nuget.org/packages/Microsoft.ML.SampleUtils)

File: dotnet/xml/ns-Microsoft.ML.Trainers.LightGbm.xml [hunk 4]
```diff
@@ -1,7 +1,10 @@
 <Namespace Name="Microsoft.ML.Trainers.LightGbm">
   <Docs>
-    <summary>Namespace containing the various <a href="https://lightgbm.readthedocs.io/en/latest/">LightGBM</a> trainers.
-      This namespace is contained in the <a href="https://www.nuget.org/packages/Microsoft.ML.LightGBM/">Microsoft.ML.LightGBM</a> nuget package. 
+    <summary>
+      Namespace containing trainers, model parameters, and utilities for <a href="https://lightgbm.readthedocs.io/en/latest/">LightGBM</a> algorithms.
     </summary>
+    <remarks>
+      Included in the Microsoft.ML.LightGbm nuget package.
```
Comment (sfilipi) [inline dotnet/xml/ns-Microsoft.ML.Trainers.LightGbm.xml]: >Microsoft.ML.LightGbm [](start = 22, length = 21)

should it link?

File: dotnet/xml/ns-Microsoft.ML.Runtime.xml (hunk 2 above)
Comment (natke) [reply dotnet/xml/ns-Microsoft.ML.Runtime.xml]: That sounds good to me. @shmoradims ?

File: dotnet/xml/ns-Microsoft.ML.Transforms.Onnx.xml [hunk 5]
```diff
@@ -1,6 +1,7 @@
 <Namespace Name="Microsoft.ML.Transforms.Onnx">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>
+      Namespace containing ONNX model loading and transformation components.
```
Comment (sfilipi) [inline dotnet/xml/ns-Microsoft.ML.Transforms.Onnx.xml]: >ONNX [](start = 27, length = 4)

link [https://onnx.ai/](https://onnx.ai/)

File: dotnet/xml/ns-Microsoft.ML.Transforms.TimeSeries.xml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <Namespace Name="Microsoft.ML.Transforms.TimeSeries">
   <Docs>
-    <summary>A set of time series transfomer, estimator and their utilities, 
-      used for <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks#anomaly-detection"> anomaly detection.</a>
+    <summary>
+        Namespace containing time-series data transformation components used for <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks#anomaly-detection">anomaly detection.</a>
```
Comment (sfilipi) [inline dotnet/xml/ns-Microsoft.ML.Transforms.TimeSeries.xml]: > used for <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks#anomaly-detection">anomaly detection.</a> [](start = 71, length = 133)

let's remove this. I think they can also do just time series analysis with them.

File: dotnet/xml/ns-Microsoft.ML.xml [hunk 7]
```diff
@@ -1,5 +1,7 @@
 <Namespace Name="Microsoft.ML">
   <Docs>
-    <summary>The main namespace for <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/">ML.NET</a></summary>
+    <summary>
+      The main namespace for <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/">ML.NET</a>. Contains application and operation contexts, transformer and trainer catalogs, and components for data view processing.
```
Comment (sfilipi) [inline dotnet/xml/ns-Microsoft.ML.xml]: >data view [](start = 207, length = 9)

should it link?

File: dotnet/xml/ns-Microsoft.ML.Transforms.TimeSeries.xml (hunk 6 above)
Comment (natke) [reply dotnet/xml/ns-Microsoft.ML.Transforms.TimeSeries.xml]: ok

File: dotnet/xml/ns-Microsoft.ML.xml (hunk 7 above)
Comment (natke) [reply dotnet/xml/ns-Microsoft.ML.xml]: We decided not to link
